spec/amp-cors-requests.md

```diff
@@ -368,6 +368,59 @@ Our response headers would be:
 Access-Control-Allow-Credentials: true
 Access-Control-Allow-Origin: https://example-com.cdn.ampproject.org
 ```
+## Working with cached fonts
+Google AMP Cache caches AMP HTML documents, images and fonts to optimize the speed of the AMP page. While making the AMP page fast, we also want to be careful in securing the cached resources. We will be making a change in how AMP cache responds it’s cached resources, 
+typically for fonts, by respecting the origin’s `Access-Control-Allow-Origin` value.
+
+### How AMP Cache works as-is
+When an AMP page was loading [https://example.com/some/font.ttf](https://example.com/some/font.ttf) from `@font-face src` attribute, AMP Cache will cache the font file and serve the resource as below with having the wild card `Access-Control-Allow-Origin`.
+
+- URL [https://example-com.cdn.ampproject.org/r/s/example.com/some/font.tff](https://example-com.cdn.ampproject.org/r/s/example.com/some/font.tff)
+- Access-Control-Allow-Origin: *
+
+### Font implementation 
```

Comment:
I didn't immediately catch on that this is documenting a behavior change. Maybe the subheadings should read:

```
### Current behavior (before October 2019)

### New behavior (October 2019 and after)
```